### ISSUE TITLE
Revise the setting of the depth network

### ISSUE THREAD

**Issue by taewookum** [zxcqlf/MonoViT#11]
For training, please download monodepth2, replace the depth network, and revise the setting of the depth network, the optimizer and learning rate according to trainer.py.

I read the contents of the sentence above. I don't know how to modify trainer.py. Can you help me?

**Comment by zxcqlf** [comment]
It is quite easy to do that, just replace the corresponding part.

**Comment by taewookum** [comment]
Do I just need to change the encoder part?  Without input?
